tests/run-make/stdin-rustdoc/rmake.rs

```diff
@@ -0,0 +1,27 @@
+//! This test checks rustdoc `-` (stdin) handling
+
+extern crate run_make_support;
+
+use run_make_support::{rustdoc, tmp_dir};
+
+const INPUT: &str = r#"
```

Comment:
```suggestion
static INPUT: &str = r#"
```

(minor) doesn't really matter tho